app/src/main/res/values/colors.xml

```diff
@@ -13,6 +13,7 @@
     <color name="mid_blue">#1E5DEA</color>
     <color name="white">#FFFFFF</color>
     <color name="black">#000000</color>
+    <color name="red">#d0021b</color>
```

Comment:
This is the same color as the `venetian_red` that appears few lines below, please use it instead of a generic `red`.